disco_aws_automation/disco_vpc.py

```diff
@@ -227,6 +229,33 @@ def _open_customer_ports(self):
                 src_security_group_group_id=self.networks["dmz"].security_group.id
             )
 
+    def _update_nat_gateways(self, network):
+        eips = self.get_config("{0}_nat_gateways".format(network.name))
```

Comment:
I think EIPs are better for long lived environments, such as in the case of mAS where we need to have static public IPs so that LAUSD can whitelist them.

Maybe we could add an option to allow for dynamic public IPs to be used, in a separate PR? ;-)